testdata/ci/features_alien_test.go

```diff
@@ -209,6 +209,70 @@ func (c *suiteContext) iRunTheWorkflowNamed(workflowName string) error {
 	return nil
 }
 
+func (c *suiteContext) iRunAsyncTheWorkflowNamed(workflowName string) error {
+	if c.applicationID == "" || c.environmentID == "" {
+		return fmt.Errorf("Missing mandatory context parameter applicationID: %q or environmentID: %q", c.applicationID, c.environmentID)
+	}
+
+	go func() {
```

Comment:
Running a routine without a way stop it is a bad practice :smile:

Reply:
fixed

Reply:
Hum I think your first through was better.
I didn't realize that RunWorkflow was synchronous and its cancellation done thanks to the context.
So as soon RunWorkflow returns for completion or cancellation then we will exit the routine.
Sorry about this.

Reply:
done. But why the previous version was better ?